pkg/schedule/schedulers/hot_region_v2.go

```diff
@@ -262,6 +268,7 @@ func (bs *balanceSolver) getScoreByPriorities(dim int, rs *rankV2Ratios) int {
 		// maxBetterRate may be less than minBetterRate, in which case a positive fraction cannot be produced.
 		minNotWorsenedRate = -bs.getMinRate(dim)
 		minBetterRate = math.Min(minBalancedRate*rs.perceivedRatio, lowRate*rs.minHotRatio)
+		minBetterRate = math.Min(minBetterRate, topnRate)
```

Comment:
why no need for the`pre-balanced state`? and should maxBetterRate be influenced by it?

Reply:
In this branch, it means that there is a big difference between a high store and a low store.
Previously, we limited the scheduling by `minBalancedRate*rs.perceivedRatio` and  `lowRate*rs.minHotRatio` to avoid scheduling a region that is too small.
However, these values are not flexible enough for the scenario involved in the issue, which is a hotspot store composed of small regions, so we use topn for evaluation.